src/valhalla_expand_bounding_box.cc

```diff
@@ -0,0 +1,112 @@
+#include "baldr/graphreader.h"
```

Comment:
this is the program that lets you get bounding boxes on the command line. this will be unfortunate for the coverage report